tests/ui_tests/app/page/service/page.py

```diff
@@ -56,24 +72,145 @@ def __init__(self, driver, base_url, cluster_id: int, service_id: int):
         self.toolbar = CommonToolbar(self.driver, self.base_url)
         self.table = CommonTableObj(self.driver, self.base_url)
 
+    @allure.step("Open Main tab by menu click")
+    def open_main_tab(self):
```

Comment:
Please specify return type since not all methods named like that return page

Reply:
add hints